src/haystack/analyzer.clj

```diff
@@ -120,17 +120,27 @@
     (flag-frame frame :repl)
     frame))
 
+(defn- tool? [frame-name last?]
+  (let [demunged (repl/demunge frame-name)]
+    (boolean (or (re-find #"^clojure\.lang\.AFn|^clojure\.lang\.RestFn|^clojure\.lang\.RT|clojure\.lang\.Compiler|^nrepl\.|^cider\.|^clojure\.core/eval|^clojure\.core/apply|^clojure\.core/with-bindings|^clojure\.core/binding-conveyor-fn|^clojure\.main/repl"
```

Comment:
I think this huge regexp can be make a `defvar` to reduce future diffs in the function and to make it possible for clients to add/change it.